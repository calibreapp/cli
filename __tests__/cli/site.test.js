const { runCLI } = require('../utils')

test('missing site argument', async () => {
  const out = await runCLI({ args: 'site pages', testForError: true })
  expect(out).toMatchSnapshot()
})

test('missing site argument value', async () => {
  const out = await runCLI({ args: 'site pages --site', testForError: true })
  expect(out).toMatchSnapshot()
})
