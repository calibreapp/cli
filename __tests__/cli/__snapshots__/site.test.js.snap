// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`missing site argument 1`] = `
"

 Missing required argument: site 

 calibre site pages [options]

List Pages for a selected Site.

Options:

  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
  --site     A unique slug identifying each Site. You can find it in Site Settin
             gs → General or by using the calibre site list command.
                                                             [string] [required]
  --count    The number of items to return (default: 25, maximum: 500).
                                                                   [default: 25]
  --cursor   The cursor to fetch records after
  --json     Outputs the results of the command in JSON format."
`;

exports[`missing site argument value 1`] = `
"

 Not enough arguments following: site 

 calibre site pages [options]

List Pages for a selected Site.

Options:

  --help     Show help                                                 [boolean]
  --version  Show version number                                       [boolean]
  --site     A unique slug identifying each Site. You can find it in Site Settin
             gs → General or by using the calibre site list command.
                                                             [string] [required]
  --count    The number of items to return (default: 25, maximum: 500).
                                                                   [default: 25]
  --cursor   The cursor to fetch records after
  --json     Outputs the results of the command in JSON format."
`;
